tf2onnx/shape_inference.py

```diff
@@ -175,7 +175,22 @@ def infer_shape_for_node(g, node):
 
         g.set_shape(node.output[0], new_shape)
         log.debug("set %s node [%s] with new shape %s", node.type, node.output[0], new_shape)
+        return True
+
+    if node.type == "ExpandDims":

```

Comment:
maybe we need start adding test for the inference.

Reply:
yep, need to think about it.